static/js/navigation.js

```diff
@@ -1,10 +1,9 @@
 /* globals $, AP */
 $(".go-back").click(function (event) {
   event.preventDefault();
-  const previousPagePath = $(event.target).data("previous-page-path");
 
-  if (previousPagePath) {
-    AP.navigator.go("addonmodule", {moduleKey: previousPagePath});
+  if (AP.history) {
```

Comment:
is AP always defined? could add extra defense with `AP && AP.history`

Reply:
In all of the pages where we load `navigation.js`, the `https://connect-cdn.atl-paas.net/all.js` has also been loaded. 
So, it shouldn't error out, but agreed with you. So, added an extra check.